deploy/playbooks/deploy_vagrant.yml

```diff
@@ -7,6 +7,7 @@
     - celery
   vars:
      app_name: "mita"
+     prod_fqdn: "local.mita.ceph.com"
```

Comment:
we might want to do something like `mita.local` so that we can use the `.local` address without interfering with `ceph.com`